translations/messages.pl.yaml

```
@@ -34,6 +34,9 @@ commerce_weavers_sylius_tpay:
                         october: 'Październik'
                         november: 'Listopad'
                         december: 'Grudzień'
+                    month_placeholder: 'Wybierz miesiąc'
+                    year: Rok
```

```suggestion
                    year: 'Rok'
```
And for messages.en.yaml it could be changed as well :)